server/test/services/applicant/ApplicantServiceTest.java

```diff
@@ -2097,6 +2097,47 @@ applicant.id, trustedIntermediaryProfile, createRequestWithFastForwardDisabled()
             result.unapplied().get(1));
   }
 
+  @Test
+  public void relevantProgramsForNoApplicant() {
```

Comment:
**question** do we need to duplicate this test in `ApplicantServiceFastForwardEnabledTest`?